application/src/main/java/org/togetherjava/tjbot/features/filesharing/FileSharingMessageListener.java

```diff
@@ -133,7 +133,7 @@ private void processAttachments(MessageReceivedEvent event,
             .build()
             .createGist()
             .public_(false)
-            .description("Uploaded by " + event.getAuthor().getAsTag());
+            .description("Uploaded by @ " + event.getAuthor().getAsTag().replace("#0000", " "));
```

Comment:
is there any reason not to use `event.getAuthor().getName()` here, you wouldn't need to replace anything.

There's still room for improvement, imo if we can retrieve user as Member, we can get the EffectiveName of user(nickname of user or username if it's null). Although i don't think `event.getMember` works here, would need to retrieve user seperately what you think @Zabuzard ?

Reply:
we shouldnt put the effective name though, cause we need to be able to find users uploading malicious content even after renaming or kicked. the account name is enough